### ISSUE TITLE
licensing

### ISSUE THREAD

**Issue by hannesm** [mirage/mirage-solo5#66]
The opam file advertises `ISC` as license, but some files - e.g. main.ml - include a LGPL 2.1 with static linking exception header.

In the meantime (June 2018, lwt 4.1.0, https://github.com/ocsigen/lwt/issues/560) lwt was relicensed to MIT. I'd propose the folliowing:
- update the copyright statement in main.ml to reflect lwt 4.1.0 license
- figure out how compatible ISC and MIT are -- eventually it makes sense to relicense mirage-solo5 as MIT?

**Comment by mato** [comment]
Yes, the licensing is a bit of a mess, some of it inherited from the old mirage-platform package and elsewhere. It would be good to clean it up, I suspect mirage-xen (old and new) has similar issues.

/cc @mirage/core

**Comment by dinosaure** [comment]
> figure out how compatible ISC and MIT are -- eventually it makes sense to relicense mirage-solo5 as MIT?

It seems that ISC license is an equivalent of MIT license (see [this Quora's response](https://qr.ae/pNPcR0)):

> The ISC license is a permissive free software license written by the Internet Systems Consortium (ISC). It is meant to be functionally equivalent to the simplified BSD and the MIT licenses, differing in its removal of language deemed unnecessary following the global adoption of the Berne Convention.
